### DISCUSSION TITLE
Error using antsApplyTransform

### DISCUSSION THREAD

**Discussion by arianayoum** [ANTsX/ANTs#1692] [Q&A] [answered]
Hello, I am trying to transform my preprocessed functional image to my template image, but I am getting the error "Segmentation fault (core dumped)". Any insight or advice would be wonderful, thank you! 

My code:

    antsApplyTransforms \
    -d 3 \
    -i sub-002_task-MID_space-T1w_desc-preproc_bold.nii.gz \
    -r colintoOAWarped_fmriprep.nii.gz \
    -o sub-002_task-MID_temp.nii.gz \
    -n nearestNeighbor \
    -t sub-002_step1_1Warp.nii.gz \
    -t sub-002_step1_Affine.txt \
    -v 1

My files were too large so I could not upload it directly here, but here's a link to a Dropbox folder that contains all 4 files: https://www.dropbox.com/scl/fo/orn4crsc02ni0merb75tl/h?rlkey=pa2337qczq3afx7yrn1tlt90r&dl=0

**Comment by cookpa** [comment, marked as answer]
You need to add `-e 3` to tell `antsApplyTransforms` to expect a moving time series image

**Reply by arianayoum** [reply]
That was silly on my part, thanks so much for catching that!

**Reply by cookpa** [reply]
No problem, this has come up many times so I'm going to add a check to help remind users #1693
